RecoVertex/V0Producer/src/V0Fitter.h

```diff
@@ -47,18 +41,28 @@
 #include "Geometry/CommonDetUnit/interface/GeomDet.h"
 #include "Geometry/TrackerGeometryBuilder/interface/GluedGeomDet.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
-
 #include "DataFormats/TrackingRecHit/interface/TrackingRecHit.h"
 #include "DataFormats/BeamSpot/interface/BeamSpot.h"
-
-#include <string>
-#include <fstream>
+#include "CommonTools/CandUtils/interface/AddFourMomenta.h"
```

Comment:
I didn't mean to suggest removal of all includes: only most of the new ones.

The diffs in this file have only one new non-trivial member change: reco::Vertex, which is already included.

All new includes, like MessageLogger, TrajectoryStateTransform, TSCBLBuilderNoMaterial don't appear to be necessary in the class interface declaration and should be moved to the implementation file.